REQUIREMENTS.md

```diff
@@ -1,25 +1,69 @@
-# Overview
+# Course prerequisites
 
-This document contains a list of expectations for people taking part to the Codess In The Classroom program.
+_**TL;DR:** here's a [checklist for workshop 1](#workshop-1-checklist)!_
 
-Accounts:
+If you're planning on attending the _Codess in the Classroom_ workshops, we'd
+like to request that you have a few things set up before you arrive. This
+will help make the workshop run smoothly, and will give us more time to talk
+about interesting topics!
 
-* You will need a free **GitHub** account. (http://github.com)
+## For all workshops
 
-* You will need a free **Microsoft Account** with a free Azure subscription (https://azure.microsoft.com/free/)
+Please make sure that you have:
 
-Development tools:
+1. a [GitHub](https://github.com) account, and
 
-* We will be using **Visual Studio Code** (https://code.visualstudio.com/) as the main IDE for the program.
+2. a [free Azure subscription](https://azure.microsoft.com/free/).
 
-  * You are free to use other tools, but we might not be able to help you during the classroom if you are
-    using a tool we don’t know how to use.
+Both of these accounts will be free to set up. We will also expect you to
+have the following development tools installed:
 
-* We will be using the **git** version control system, so you should be familiar with its basics.
-  * Suggested tutorial: https://learngitbranching.js.org/ ("Introduction Sequence")
+3. A programmers' text editor such as [Visual Studio
+   Code](https://code.visualstudio.com/).
 
-Programming languages
+   If you are familiar with another editor, feel free to use it! If you do
+   so, we can't guarantee that we'll be able to help you with it during the
+   workshop (but we'll do our best!).
 
-* Basic familiarity with **JavaScript** is required, because we’ll be developing a simple web front-end
+4. The [Git](https://git-scm.com/) distributed version control system
+   software.
 
-* Familiarity with a language chosen among **C#**, **JavaScript**, **Python** or **Java** is required for the application back-end
```

Comment:
I'm assuming you dropped this on purpose, and want to re-introduce closer to the appropriate workshop, right?

Reply:
👍